src/tactic/command/global_search_trigger.py

```diff
@@ -104,7 +144,207 @@ def cleanup(my, data):
         #is_ascii = my.is_ascii(data)
         return Common.extract_keywords(data)
      
+    def set_searchable_keywords(my, sobj):
+        '''
+        Used to set the searchable_keywords column. Reads from the keywords_data
+        column.
+        '''
+        if not sobj.column_exists("searchable_keywords"):
+            return
+
+        keywords_data = sobj.get_json_value("keywords_data", {})
+
+        if keywords_data:
+            path = ""
+            if 'path' in keywords_data:
+                path = keywords_data['path']
+
+            user = ""
+            if 'user' in keywords_data:
+                user = keywords_data['user']
+
+            collection = ""
+            if 'collection' in keywords_data:
+                
+                collection_keywords_data = keywords_data['collection']
+                for collection_code in collection_keywords_data.keys():
+                    collection = collection + " " + collection_keywords_data[collection_code]
```

Comment:
don't use +  in string concatenation.. keep it as a list and at the next line u can do 
" ".join(collection_list)